ompi/mca/pml/ob1/pml_ob1_recvreq.c

```diff
@@ -1373,18 +1376,13 @@ void mca_pml_ob1_recv_req_start(mca_pml_ob1_recv_request_t *req)
         OB1_MATCHING_UNLOCK(&ob1_comm->matching_lock);
     } else {
         if(OPAL_LIKELY(!IS_PROB_REQ(req))) {
-            PERUSE_TRACE_COMM_EVENT(PERUSE_COMM_REQ_MATCH_UNEX,
-                                    &(req->req_recv.req_base), PERUSE_RECV);
-
             hdr = (mca_pml_ob1_hdr_t*)frag->segments->seg_addr.pval;
-            PERUSE_TRACE_MSG_EVENT(PERUSE_COMM_MSG_REMOVE_FROM_UNEX_Q,
-                                   req->req_recv.req_base.req_comm,
-                                   hdr->hdr_match.hdr_src,
-                                   hdr->hdr_match.hdr_tag,
-                                   PERUSE_RECV);
 
-            PERUSE_TRACE_COMM_EVENT(PERUSE_COMM_SEARCH_UNEX_Q_END,
-                                    &(req->req_recv.req_base), PERUSE_RECV);
+            MCA_BASE_EVENT_RAISE (mca_pml_ob1_events[MCA_PML_OB1_EVENT_UNEX_REMOVE].event,
```

Comment:
Just to be sure: there is no equivalent to PERUSE REQ_MATCH_UNEX ?

Reply:
There is not an implemented equivalent.  In this case, MCA_PML_OB1_EVENT_UNEX_REMOVE functions as a similar event, as it is only used here.  Do you think it would be good to add a specific match event?

Reply:
To compute the duration of the unexpected search one needs to first know if the search was successful or not. If successful then the duration should be computed as `MCA_PML_OB1_EVENT_UNEX_REMOVE - MCA_PML_OB1_EVENT_SEARCH_UNEX_BEGIN` while if the search was not successful it should be computed as `MCA_PML_OB1_EVENT_SEARCH_UNEX_END - MCA_PML_OB1_EVENT_SEARCH_UNEX_BEGIN`. As long as it is well documented, it should be good.